src/app.js

```diff
@@ -0,0 +1,37 @@
+import { productData } from './productData.js';
```

Comment:
this shouldn't be in brackets because in your productData.js you used an  export default.